clients/drcachesim/tracer/raw2trace.cpp

```diff
@@ -1404,6 +1412,15 @@ raw2trace_t::get_file_type(void *tls)
     return tdata->file_type;
 }
 
+void
+raw2trace_t::set_file_type(void *tls, offline_file_type_t file_type)
+{
+    // The file type needs to be updated during the switch to correctly process the
+    // entries that follow after. This does not affect the written-out type.
```

Comment:
s/that follow after/that follow after TRACE_MARKER_TYPE_FILTER_ENDPOINT/, to provide some context here.

Reply:
Done.